llvm/lib/Support/CommandLine.cpp

```diff
@@ -2734,6 +2734,39 @@ void cl::PrintHelpMessage(bool Hidden, bool Categorized) {
     CommonOptions->CategorizedHiddenPrinter.printHelp();
 }
 
+ArrayRef<StringRef> cl::CompilerBuildConfig = {
+#if LLVM_IS_DEBUG_BUILD
+    "+unoptimized",
+#endif
+#ifndef NDEBUG
+    "+assertions",
+#endif
+#ifdef EXPENSIVE_CHECKS
+    "+expensive-checks",
+#endif
+#if __has_feature(address_sanitizer)
+    "+asan",
+#endif
+#if __has_feature(undefined_behavior_sanitizer)
```

Comment:
sort sanitizers alphabetically?

You can add thread_sanitizer and `hwaddress_sanitizer` as well

Reply:
Good catch on TSan.
I followed the documented list of `__has_feature` that I could see, I don't see `hwaddress_sanitizer` in the list here: https://clang.llvm.org/docs/LanguageExtensions.html#extensions-for-dynamic-analysis